deployment/values.yaml

```diff
@@ -37,13 +37,21 @@ arguments: ["-f", "/etc/dcgm-exporter/dcp-metrics-included.csv"]
 # the watched exporter fields.
 # Example arguments: ["-m", "default:exporter-metrics-config-map"]
 
+# Image pull secrets for container images
 imagePullSecrets: []
+
+# Overrides the chart's name
 nameOverride: ""
+
+# Overrides the chart's computed fullname
 fullnameOverride: ""
```

Comment:
Isn't this app name that assumes chart name by default?

Reply:
According to the helm: default fully qualified app name. See: https://github.com/NVIDIA/dcgm-exporter/blob/main/deployment/templates/_helpers.tpl#L10

Reply:
Right, thats why I thought it should read "Overrides app name", since this is the name that would show up on the deamonset, SA and RBACs.

Nevermind, its already merged.